backend/app/crud/tts_evaluations/cron.py

```
@@ -0,0 +1,546 @@
+                    wav_data = pcm_to_wav(pcm_data)
+
+                    # Calculate duration
+                    duration = calculate_duration(len(pcm_data))
+
+                    # Upload WAV to S3
+                    audio_filename = f"{uuid.uuid4()}.wav"
+                    audio_url = upload_to_object_store(
+                        storage=storage,
+                        content=wav_data,
+                        filename=audio_filename,
+                        subdirectory=f"evaluations/tts/audio",
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)